nussl/core/effects.py

```diff
@@ -159,8 +159,9 @@ def apply_effects_sox(audio_signal, filters):
         input_array=np.transpose(audio_data), 
         sample_rate_in=audio_signal.sample_rate
     ) 
-    
-    augmented_signal = AudioSignal(audio_data_array=np.transpose(augmented_data))
+
+    augmented_signal = AudioSignal(audio_data_array=np.transpose(augmented_data),
```

Comment:
Is there a solution that uses `make_copy_with_audio_data` instead? If done this way then other properties like `stft_params` won't propagate either, I think.

Reply:
Also, is the `np.transpose` redundant? We check to make sure the shape is correct whenever `AudioSignal.audio_data` is set, so you can clean up this line by removing the transpose.

Reply:
Oh yeah good point. Also Andreas use `x.T`, not `np.transpose`, I think. Just a style thing.

Reply:
would we want to make a copy of the audio signal for ffmpeg effects as well? right now, the label nor stft is cascaded for ffmpeg

Reply:
I think that would be good, yes. Add some test cases for both as well - thanks!